src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java

```diff
@@ -102,7 +103,7 @@ public String getNormalName(String netherName, PortalType type) {
             }
         }
         if (!normalName.equals(netherName) && plugin.getCore().getMVWorldManager().isMVWorld(normalName)) {
-            this.plugin.log(Level.FINEST,  "Selected normal world '" + normalName + "' for " + type + " '" + netherName + "'");
+			Logging.finest("Selected normal world '" + normalName + "' for " + type + " '" + netherName + "'");
```

Comment:
Same as above.

Reply:
can seem to change it... is intelliJ being dumb lol

Reply:
should be fixed lol idk why the entire file was in tabs not spaces.